### DISCUSSION TITLE
How to completely reinstall Docker Wyze Bridge in Home Assistant

### DISCUSSION THREAD

**Discussion by rsilves** [mrlt8/docker-wyze-bridge#1265] [General]
Hello! I have been using the Docker Wyze Bridge Home Assistant Add-on for almost a year successfully, and this week I needed to re-position my cameras and rename them, and the Docker Wyze Bridge didn’t update the names correctly, nor the feeds. I thought I could uninstall it and reinstall it, but I was wrong: after reinstalling it, it doesn’t ask for any kind of login info, it just shows the same as before (the old camera layout, with the old names and thumbnails). I also tried removing the repository and adding it back, and also deleting the API key and creating a new one. No luck, it seems stuck.
I also tried setting my new API key in the "Show unused optional configuration options" fields (the previous login was done through the web UI), again with no success at all. Of course I tried restarting the add-on and restarting the Home Assistant Yellow completely.
What can I do to completely uninstall and reinstall this add-on?
Thanks!
Rodrigo

**Comment by mrlt8** [comment]
You can probably set fresh_data to true.

**Reply by rsilves** [reply]
Thanks @mrlt8, this solved the issue. Should I keep it on true or is it a one time thing?

**Reply by mrlt8** [reply]
Setting it to false will reduce the number of calls to the wyze servers, but it shouldn't really matter either way.

**Comment by tbsmith1** [comment]
I hope it is okay to add to this thread after so long.  I deleted the app (I had installed it as an add on in HA) because I could never get it to work.  But there are still remnants that I can see in MQTT Explorer.  Is there any way to delete them permanently?  I didn't see anything in the  documentation.  Thanks.

<img width="337" height="262" alt="image" src="https://github.com/user-attachments/assets/b3ed0d0f-304b-4b70-ad2c-f8c632778207" />

I beleive it is completly removed as it doesn't show up in any of my disabled apps or integrations, and I don't see it in any docker containers (assuming I am doing that right):

<img width="943" height="803" alt="image" src="https://github.com/user-attachments/assets/ade4f065-ecfc-44e6-bf45-407b0024b373" />

**Reply by tbsmith1** [reply]
My apologies, it was as simple as hitting the trash icon in MQTT Explorer!
